docs/sources/tempo/troubleshooting/long-running-traces.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Long-running traces
+description: Troubleshoot search results when using long-running traces
+weight: 479
+aliases:
+  - ../operations/troubleshooting/long-running-traces/
+---
+
+# Long-running traces
+
+An issue arises in Tempo when a user exercises a usage pattern referred to as
+long-running traces. This happens when Tempo receives spans for a trace,
+followed by a delay, and then Tempo receives additional spans for the same
+trace. If the delay between spans is great enough, the spans end up in
+different blocks, which can lead to inconsistency in a few ways.
```

Comment:
Trying to rewrite to reduce passive voice. What do you think about this as a start? 
```suggestion
Long-running traces are created when Tempo receives spans for a trace,
followed by a delay, and then Tempo receives additional spans for the same
trace. If the delay between spans is great enough, the spans end up in
different blocks, which can lead to inconsistency in a few ways:
```